### PR TITLE
[RW-7034]: use AccessModuels for AAR and remove use of ReneableAccessModules

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:
I verified dual write worikng by query like:
https://docs.google.com/document/d/1V3V5k-cRfEcwScPMs9AIeNDyJieDdk-vZUszCQ2pZaY/edit#

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by jmthibault79 — APPROVED** [review]
Works great.  Amazing how simple a switch this is.  Planning ahead with the structure of the new Access Modules really paid off.

File: ui/src/app/utils/access-utils.spec.tsx [hunk 1]
```diff
@@ -24,73 +23,65 @@ const todayPlusDays = (days: number): number => {
...
+    }, {
+      moduleName: AccessModule.DATAUSERCODEOFCONDUCT,
       expirationEpochMillis: todayPlusDays(31),
     }]
   }
 }
 
 describe('maybeDaysRemaining', () => {
-  it('returns undefined when the profile has no renewableAccessModules', () => {
+  it('returns undefined when the profile has no accessModules', () => {
```
[the remark this answers is not in the document]
Comment (yonghaoy) [reply ui/src/app/utils/access-utils.spec.tsx:76]: that is the line 80

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/utils/access-utils.spec.tsx:76]: what I mean is: include a module which is not on the AAR page

[the remark this answers is not in the document]
Comment (yonghaoy) [reply ui/src/app/utils/access-utils.spec.tsx:76]: Sorry, I misunderstood expirable/expired. 
Here it is: https://github.com/all-of-us/workbench/pull/5392

**Pull request merged**